### ISSUE TITLE
test_quantity_array_methods error with numpy release candicate

### ISSUE THREAD

**Issue by olebole** [astropy/astropy#3471]
If the numpy version number has an "rc" in it, some tests give errors because of:
```
_____ ERROR collecting astropy/units/tests/test_quantity_array_methods.py ______
astropy/units/tests/test_quantity_array_methods.py:11: in <module>
>   NUMPY_LT_1P9P1 = [int(x) for x in np.__version__.split('.')[:3]] < [1, 9, 1]
E   ValueError: invalid literal for int() with base 10: '0rc1'
```
Here, Numpy is 1.9.0rc1, Platform is Debian (inofficial) on x32. [Full log](http://buildd.debian-ports.org/status/fetch.php?pkg=python-astropy&arch=x32&ver=1.0~rc1-3&stamp=1423526976)

**Comment by embray** [comment]
Thanks--this should probably be fixed by #3389.  I was holding back on merging that as it seemed non-critical.  But now that there is an actual bug case (as I was worried their might be when I saw that same code, though I had never directly encountered an issue with it) it might be worth merging my fix.

**Comment by eteq** [comment]
@olebole - can you check that #3389 fixed this, and close this if so?  Thanks!

**Comment by olebole** [comment]
Sorry, I don't have access to an x32 machine unti tomorrow evening, and the change seems too complicated to just test it somewhere else.

**Comment by embray** [comment]
It should be fixed, in any case.  The code that's bugging out there simply no longer exists, and this version comparison is handled correctly by `pkg_resources`:
```python
>>> from pkg_resources import parse_version
>>> parse_version('1.9.0rc1') < parse_version('1.9.1')
True
```

**Comment by olebole** [comment]
OK, so I am closing this. If this reappears with 1.0, I will just re-open.

**Comment by embray** [comment]
I went ahead and reported this as fixed in f04c3a1, but I will roll that back if it turns out not to be somehow.

**Comment by embray** [comment]
Okay, thanks.
